style.css

```diff
@@ -0,0 +1,832 @@
+}
+
+#program h4 {
+  color: white;
+  text-align: center;
+  margin-top: 7%;
+  background: url(Images/underline.png) bottom no-repeat;
+  padding-bottom: 10px;
+  background-size: 10%;
+}
+
+.logo {
+  padding-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 20px;
+}
+
+#Creators {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+}
+
+.creators1,
+.creators2 {
+  display: flex;
+  flex-direction: column;
+  margin-inline: 2.5%;
+  row-gap: 50px;
+  padding-top: 10%;
+}
+
+.creators2 {
+  padding-bottom: 10%;
+}
+
+.card,
+.carh {
+  display: flex;
+  height: auto;
+  column-gap: 5%;
+}
+
+.creators2,
+.cardh {
+  display: none;
+  column-gap: 5%;
+}
+
+.creator-img {
+  width: 25%;
+  height: 40%;
+}
+
+#CContent {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.card p,
+.card h3,
+.cardh p,
+.cardh h3 {
+  font-family: 'Lato', sans-serif;
+}
+
+.card h3,
+.cardh h3 {
+  color: #272a31;
+  font-weight: 900;
+}
+
+.card p,
+.cardh p {
+  font-size: small;
+}
+
+.button {
+  display: flex;
+  justify-content: center;
+}
+
+.btn2 {
+  background-color: white;
+  border: #d1d1d1 1px solid;
+  padding: 10px 45%;
+  margin-bottom: 30px;
+  background-image: url(Images/Icons/more-arrow.png);
+  background-repeat: no-repeat;
+  background-position: top 50% right 40%;
+  background-size: 5%;
+}
+
+.logo p {
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-size: small;
+}
+
+.logo img {
+  width: 80%;
+  border: #d1d1d1 1px solid;
+  padding: 10px 10px;
+}
+
+.past {
+  padding-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 20px;
+  border-top: #d1d1d1 1px solid;
+  row-gap: 10px;
+}
+
+.past p {
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+}
+
+.past h3 {
+  text-align: center;
+  color: white;
+  font-size: xx-large;
+  font-family: 'CocoGoose', sans-serif;
+}
+
+.old1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(Images/lbaa1.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgba(236, 82, 66, 0.74);
+  height: 200px;
+  color: white;
+}
+
+.old2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-image: url(Images/lbaa2.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgba(236, 82, 66, 0.74);
+  height: 200px;
+  color: white;
+}
+
+.oldcont {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 80%;
+  row-gap: 15px;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  row-gap: 0;
+}
+
+.partners {
+  background-color: #272a31;
+  width: 100%;
+  height: 300px;
+  background-image: url(Images/Partners/partners.jpg);
+  background-repeat: no-repeat;
+  background-size: auto 100%;
+  background-position: top left 50%;
+}
+
```

Comment:
- Kindly ensure your partner's icons in the desktop version are placed horizontally to match the design

*your design*
![horizontally](https://user-images.githubusercontent.com/59397159/181862621-d924ccec-f867-4961-9714-c42fc4b3df63.png)


*main design*
![here](https://user-images.githubusercontent.com/59397159/181862609-df7e0917-d0bf-40a5-bb7a-d7474451dd4a.png)